vms/platformvm/vm_test.go

```diff
@@ -356,6 +517,140 @@ func TestGenesis(t *testing.T) {
 	}
 }
 
+func TestGenesisGetUTXOs(t *testing.T) {
+
+	addr := keys[0].PublicKey().Address()
+	hrp := constants.NetworkIDToHRP[testNetworkID]
+	addrString, _ := formatting.FormatBech32(hrp, addr.Bytes())
+
+	// Create a starting point of 1100 UTXOs
+	utxoCount := 1100
+	var genesisUTXOs []APIUTXO
+	for i := 0; i < utxoCount; i++ {
+		genesisUTXOs = append(genesisUTXOs, APIUTXO{
+			Amount:  json.Uint64(defaultBalance),
+			Address: addrString,
+		})
+	}
+
+	// Inject them in the Genesis build
+	buildGenesisArgs := BuildGenesisArgs{
+		NetworkID:     json.Uint32(testNetworkID),
+		AvaxAssetID:   avaxAssetID,
+		UTXOs:         genesisUTXOs,
+		Validators:    []APIPrimaryValidator{},
+		Chains:        nil,
+		Time:          json.Uint64(defaultGenesisTime.Unix()),
+		InitialSupply: json.Uint64(360 * units.MegaAvax),
+		Encoding:      formatting.HexEncoding,
+	}
+
+	_, _, vm, _ := GenesisVMWithArgs(t, &buildGenesisArgs)
+
+	addrsSet := ids.ShortSet{}
+	addrsSet.Add(addr)
+
+	// Fetch all UTXOs
+	utxosNonPaginated, _, _, err := vm.GetUTXOs(vm.DB, addrsSet, ids.ShortEmpty, ids.Empty, -1, false)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if len(utxosNonPaginated) != utxoCount {
+		t.Fatalf("Wrong number of utxos. Expected (%d) returned (%d)", utxoCount, len(utxosNonPaginated))
+	}
+
+	// First Page - using paginated calls
+	paginatedUtxos, lastAddr, lastIdx, err := vm.GetUTXOs(vm.DB, addrsSet, ids.ShortEmpty, ids.Empty, -1, true)
```

Comment:
Could you change to `paginatedUTXOs` to follow the convention of using the same case for the word utxos?

Reply:
Can do 👍